### ISSUE TITLE
Trigger Add Bookmark button from keyboard

### ISSUE THREAD

**Issue by roosterboy** [kristofa/bookmarker_for_pinboard#10]
For faster bookmarking using the extension, set the Return key to trigger the Add Bookmark button in the popover window. In combination with Issue #3 and with the text input focus now in the Tags field (issue #9), this would allow a complete bookmarking workflow without having to take one's fingers off the keyboard.

**Comment by kristofa** [comment]
Thanks @roosterboy I like the idea but couldn't find yet how to assign a keyboard shortcut that triggers the toolbar button. If you know and can share that would be really helpful.

**Comment by apas** [comment]
> couldn't find yet how to assign a keyboard shortcut that triggers the toolbar button

@kristofa, are you referring to the bookmarker extension icon on the toolbar or the `Add Bookmark` button in the pop-up? As said in #3, Instapaper supports keyboard shortcuts (see attached) and 1Password works with <kbd>cmd</kbd>+<kbd>/</kbd>.

![image](https://user-images.githubusercontent.com/347450/55781641-f140b280-5aaa-11e9-8132-9008fbfebd74.png)

**Comment by kristofa** [comment]
@apas what I meant is opening the popover / pop-up window using a keyboard shortcut, so same as if the toolbar button was clicked.  I asked the question in the Apple Developer Forum in the section about developing Safari extensions. Let's hope someone can give me some more information.

Thanks for the screenshot that shows the configuration items for the Instapaper extension. I hadn't seen that before and have no idea how to implement this. I found the documentation on how to write new Safari App Extensions still quite limited and this is my first native macOS software project :)  But I'll keep looking.

**Comment by apas** [comment]
Perhaps these links would help:

* https://stackoverflow.com/questions/4059806/how-can-i-add-hotkeys-with-safari-extension/5099132#5099132 (author links to two files on GitHub)
* http://antrikshy.com/blog/safari-extension-development-keyboard-shortcuts

They're from 2011 and 2014 respectively but you never know -- perhaps Apple didn't deprecate this API.

Edit: I think this is how Evernote's extension does it when ` is pressed -- granted their entire extension is a JS popover so I'm not sure if it's relevant to a native popup menu.

![image](https://user-images.githubusercontent.com/347450/56046829-6b3f9880-5d44-11e9-984d-01f31c82f67e.png)

**Comment by kristofa** [comment]
Sorry @roosterboy and @apas I realise that I mistakenly saw this issue as a duplicate for #3 which it clearly isn't.

Good news however is that I implemented this request and I just submitted the new version to the App Store for review. As of version 1.3 you will be able to submit the bookmark when pressing enter in the Description or Tags text fields. No need anymore to go and push the 'Add bookmark' button.
